website/src/pages/docs/migration-guides/from-graphql-cli.mdx

```diff
@@ -0,0 +1,146 @@
+import { Callout, PackageCmd } from '@theguild/components'
+
+# Migrating from GraphQL-CLI to Inspector CLI
+
+If you're currently using GraphQL-CLI in your GraphQL workflow, you may have noticed that it's no
+longer being actively maintained. As such, it's a good idea to start thinking about migrating to a
+newer tool that's actively maintained and supported. One such tool is Inspector CLI, which provides
+similar functionality to GraphQL-CLI with some additional features.
+
+<Callout type="warning">The package @graphql-cli is deprecated, and no longer maintained!</Callout>
+
+## Introduction
+
+Inspector CLI is a powerful tool that allows you to analyze and manipulate GraphQL schemas. It
+provides a number of useful actions, including `common`, `coverage`, `diff`, `introspect`,
+`similar`, and `validate`. In this post, we'll explore how to migrate from GraphQL-CLI to Inspector
+CLI and how to use the different actions.
+
+## Migration Steps
+
+The migration from GraphQL-CLI to Inspector CLI is relatively straightforward. Here are the steps
+you can take:
+
+1. Install Inspector CLI: You can install Inspector CLI using NPM by running the following command:
+
+<PackageCmd packages={[{ name: '@graphql-inspector/cli graphql', isGlobal: true }]} />
+
+2. Replace graphql-cli with graphql-inspector in your commands: After installing Inspector CLI,
+   you'll need to replace all instances of graphql in your commands with graphql-inspector.
+
+For example, if you were previously using the following command to validate your schema with
+GraphQL-CLI:
+
+```
+graphql validate './documents/*.graphql' http://localhost:3000/graphql
+```
+
+You would now use the following command with Inspector CLI:
+
+```
+graphql-inspector validate './documents/*.graphql' http://localhost:3000/graphql
+```
+
+## Examples
+
+Here are examples of how to perform some common actions with both GraphQL-CLI and Inspector CLI:
+
+### Introspect
+
+GraphQL-CLI:
```

Comment:
I think we can use Tabs here? with "Before (`graphql-cli`)" and "After (`graphql-inspector`)"